### ISSUE TITLE
NullPointerException running checksum:dependencies with test scope dependencies.

### ISSUE THREAD

**Issue by chuckdumont** [nicoulaj/checksum-maven-plugin#97]
I get the error below when running `mvn checksum:dependencies`.  If I remove my dependencies that specify `<scope>test</scope>`, then the problem goes away.  I'm using version 1.9

```
Caused by: java.lang.NullPointerException
    at net.nicoulaj.maven.plugins.checksum.digest.MessageDigestFileDigester.calculate (MessageDigestFileDigester.java:81)
    at net.nicoulaj.maven.plugins.checksum.execution.FailOnErrorExecution.run (FailOnErrorExecution.java:72)
    at net.nicoulaj.maven.plugins.checksum.mojo.AbstractChecksumMojo.execute (AbstractChecksumMojo.java:226)
    at net.nicoulaj.maven.plugins.checksum.mojo.DependenciesMojo.execute (DependenciesMojo.java:35)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:208)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:154)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:146)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:290)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:194)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:289)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:229)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:415)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:356)
```

**Comment by chuckdumont** [comment]
I was able to work around this by using the `scopes` configuration option to ignore test scope dependencies.

I've create a pull request that fixes this so the workaround isn't needed.  #98

**Comment by bondolo** [comment]
Looks good to me.

**Comment by nicoulaj** [comment]
fixed by #98
